sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/inprocess/InProcessPipelineRunner.java

```diff
@@ -152,6 +165,12 @@
   ////////////////////////////////////////////////////////////////////////////////////////////////
   private final InProcessPipelineOptions options;
 
+  public static InProcessPipelineRunner createForTest() {
+    InProcessPipelineOptions options = PipelineOptionsFactory.as(InProcessPipelineOptions.class);
+    options.setBlockOnRun(true);
```

Comment:
This is actually not useful; the way this is created for test is via "fromOptions()", but with the InProcessPipelineRunner as the pipeline runner.

rm'd